proposals/20220823-huggingface_model_pusher.md

```diff
@@ -12,25 +12,48 @@
 **Project name:** HuggingFace Model Pusher
 
 ## Project Description
-HuggingFace Model Pusher(`HFModelPusher`) pushes blessed model to the [HuggingFace Model Hub](https://huggingface.co/models).
+HuggingFace Model Pusher(`HFModelPusher`) pushes blessed model to the [HuggingFace Model Hub](https://huggingface.co/models). Also, it optionally pushes application to [HuggingFace Space Hub](https://huggingface.co/spaces).
 
 ## Project Category
 Component
 
 ## Project Use-Case(s)
-The HuggingFace Model Hub lets us have [Git-LFS](https://git-lfs.github.com) enabled repositories in public and private modes. Supported models hosted on the HuggingFace Model Hub can be directly loaded/used with APIs provided by [transformers](https://huggingface.co/docs/transformers/index) package. However, it is not limited. We can host arbitrary types of models too. 
+The HuggingFace Model and Space Hubs let us have [Git-LFS](https://git-lfs.github.com) enabled repositories in public and private modes, and they are easy to manage versions sepecially for those familiar with Git. 
 
-HuggingFace Model Hub is easy to manage model versions, especially for those familiar with Git.
+Supported models hosted on the HuggingFace Model Hub can be directly loaded/used with APIs provided by [transformers](https://huggingface.co/docs/transformers/index) package. However, it is not limited. We can host arbitrary types of models too. 
+
+HuggingFace Space Hub comes with free resources to host prototype applications that use machine learning models. Currently supported application frameworks are Gradio and Streamlit. It is often a good idea to host current version of the model to the Huggingface Space, so it could be interated with real world before the production deployment. 
```

Comment:
```suggestion
HuggingFace Space Hub provides free resources to host prototype applications that use machine learning models. It supports both private and public modes. Currently supported application frameworks are Gradio and Streamlit. It is often a good idea to host the current version of the model to the HuggingFace Space, so it can be tested before the production deployment. But you can host your models HuggingFace Spaces for other purposes as well.
```